shaman/models/projects.py

```diff
@@ -0,0 +1,31 @@
+from sqlalchemy import Column, Integer, String
+from sqlalchemy.orm.exc import DetachedInstanceError
+from shaman.models import Base
+
+
+class Project(Base):
+
+    __tablename__ = 'projects'
+    id = Column(Integer, primary_key=True)
+    name = Column(String(256), nullable=False, unique=True, index=True)
+
+    def __init__(self, name):
+        self.name = name
+
+    def __repr__(self):
+        try:
+            return '<Project %r>' % self.name
+        except DetachedInstanceError:
+            return '<Project detached>'
+
+    def __json__(self):
+        json_ = {}
+        json_["name"] = self.name
+        return json_
```

Comment:
can't we just return the dict with `self.name` like:

    return dict(
        name=self.name,
    )

Why the back and forth?